tests/test.wake.template

```diff
@@ -0,0 +1,12 @@
+global def runAPIFIRRTLSiFiveTests =
+  def jars = scalaModuleClasspath firrtlScalaModule
+  def topName = 'AndGate'
+  def targetDir = "{here}/build"
+  def inputFile = (source "{here}/AndGate.fir")
+
+  def compilePlan = makeFirrtlCompilePlan jars topName targetDir inputFile
+  def outputs = runFirrtlCompile compilePlan
+
+  outputs.getFirrtlCompileOutputsAllOutputs
+  | map getPathResult
+  | findFail
```

Comment:
just found out that there is a `findFailFn` function that you can use like `| findFailFn getPathResult`

Reply:
Oh, nice, thanks! It'd be neat to have a Wake linter/code autocorrector that catches things like this and suggests improvements to them.

Just amended my commit to use that instead. Will merge if the tests pass.